ofl/josefinsans/METADATA.pb

```diff
@@ -10,7 +10,7 @@ fonts {
   filename: "JosefinSans[wght].ttf"
   post_script_name: "JosefinSans-Thin"
   full_name: "Josefin Sans Thin"
-  copyright: "Copyright 2010 The Josefin Sans Project Authors (https://github.com/ThomasJockin/JosefinSansFont-master), with Reserved Font Name \"Josefin Sans\"."
```

Comment:
There's nothing wrong with the original line. Please run `gftools add-font` on this family again.

Reply:
Fontbakery lists a 'FAIL' when the escape character is used (https://github.com/googlefonts/fontbakery/issues/2932). So this change removes the conflict of `"` and `'` so that escape characters aren't necessary anymore.